iocore/net/quic/QUICTLS_openssl.cc

```diff
@@ -31,11 +29,21 @@
 #include <openssl/evp.h>
 
 #include "QUICConfig.h"
-
+#include "QUICGlobals.h"
 #include "QUICDebugNames.h"
+#include "QUICPacketProtectionKeyInfo.h"
 
 static constexpr char tag[] = "quic_tls";
 
+using namespace std::literals;
+
+static constexpr std::string_view QUIC_CLIENT_EARLY_TRAFFIC_SECRET("QUIC_CLIENT_EARLY_TRAFFIC_SECRET"sv);
```

Comment:
Can you add "_TITLE" or some postfix? The name looks like a secret data.

Reply:
I added `_LABEL` as postfix. Because NSS Key Log Format call the field label.